terraform/database.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_db_instance" "default" {
+  allocated_storage    = 20
+  db_name              = var.DB_NAME
+  engine               = "postgres"
+  engine_version       = "16.1"
+  instance_class       = "db.t3.micro"
+  publicly_accessible          = true
```

Comment:
do we want it to be publicly accessible?

Reply:
At the moment, the RDS instance is publicly accessible

Reply:
Okay cool just checking!